DuckDuckGo/NewTabPage/ContinueSetUpModel+NewTabPage.swift

```diff
@@ -62,6 +62,7 @@ extension HomePage.Models.ContinueSetUpModel: NewTabPageNextStepsCardsProviding
         guard cards.contains(.addAppToDockMac) else {
             return
         }
+        continueSetUpCardsViewDidAppear()
```

Comment:
I hope you don't mind me dropping in this small change here. I forgot to connect "cards will appear" event to the model to count days of presentation.